iroh-rpc-client/src/status.rs

```diff
@@ -2,32 +2,37 @@ use crate::{gateway, network, store};
 use anyhow::Result;
 
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub enum ServiceStatus {
-    ///  Indicates rpc server is in an unknown state
+pub enum StatusType {
+    /// Indicates service status is unknown
     Unknown,
     /// Indicates service is serving data
     Serving,
-    /// Indicates service is not serving data, but the rpc server is not down
-    NotServing,
```

Comment:
Not for this PR, but with this we have the ability for the service to actively signal that it is not feeling well.

Reply:
Trying to find the balance of "keep it in for the future" and "take it out because it's not used"

I think this enum variant is a candidate for "keep it in for the future"